apps/emqx_exhook/etc/emqx_exhook.conf

```diff
@@ -24,6 +24,11 @@
 ## Value: false | Duration
 #exhook.auto_reconnect = 60s
 
+## The exhook exec priority on emqx hooks
+##
+## Default: 0
+## Value: Integer
+#exhook.hook_priority = 0
```

Comment:
`default_hook_priority`.

Reply:
It has a default value `0`.
Do we need name it with a prifix `default_` ?

Reply:
OR: add a comment to clarify this is the default hook priority for ALL the hookpoints used by this plugin.